pkg/metacni/metacni.go

```diff
@@ -521,10 +522,18 @@ func DeleteInterfaces(args *skel.CmdArgs) error {
     log.Println("INFO: DEL: DanmEp REST client could not be created because" + err.Error())
     return nil
   }
-  eplist, err := danmep.FindByCid(danmClient, cniArgs.containerId)
-  if err != nil {
-    log.Println("INFO: DEL: Could not interrogate DanmEps from K8s API server because" + err.Error())
-    return nil
+  var eplist []danmtypes.DanmEp
+  for i := 0; ; i++ {
+    eplist, err = danmep.FindByCid(danmClient, cniArgs.containerId)
+    if err == nil {
+      break
+    }
+    if i >= 10 {
```

Comment:
10X2=20 sec is insanely long time to wait, and would introduce big performance bottlenecks
I would prefer max 0.5-1 sec max, so let's say 5-10X100ms

also please use/re-use package level constants for the duration (retry count and time duration too)